src/api/firebase.js

```diff
@@ -184,10 +189,29 @@ export async function updateItem(listPath, itemId) {
 	const itemDocumentRef = doc(listCollectionRef, itemId);
 	const item = await getDoc(itemDocumentRef);
 	const itemTotalPurchases = item.data().totalPurchases;
+	const dateLastPurchased = item.data().dateLastPurchased?.toDate();
+	const dateCreated = item.data().dateCreated.toDate();
+	const dateNextPurchased = item.data().dateNextPurchased.toDate();
+	const now = new Date();
+	const daysSinceLastPurchase = getDaysBetweenDates(
+		now,
+		dateLastPurchased ? dateLastPurchased : dateCreated,
+	);
+	const prevEstimate = dateLastPurchased
+		? getDaysBetweenDates(dateNextPurchased, dateLastPurchased)
+		: undefined;
```

Comment:
Thanks for your work! 
I have a question regarding lines 200-202: In place of undefined, I would have added a calculation getDaysBetweenDates(dateNextPurchased, dateCreated), to get the initial count of days before the first purchase. What is the reason behind setting it to undefined?
@borjaMarti @BikeMouse

Reply:
Hey Céline!

You are totally right, we updated the implementation to take into account that period instead of passing undefined as the default.

Thanks for the feedback! 😄